erpnext/hr/doctype/salary_slip/salary_slip.py

```diff
@@ -1021,6 +1028,11 @@ def set_loan_repayment(self):
 				'interest_income_account': loan.interest_income_account,
 				'loan_repayment_detail': loan.loan_repayment_detail,
 				'loan_repayment_date': loan.payment_date,
+				'loan_type': loan.loan_type,
+				'disbursement_date': loan.disbursement_date,
+				'total_loan_amount': loan.total_loan_amount,
+				'total_amount_paid': flt(loan.total_amount_paid) + flt(loan.total_payment),
```

Comment:
will be changed to 
pending_loan_amount: loan.total_loan_amount - loan.total_amount_paid